test/helpers/ProposalTest.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+import {
+  ICompoundTimelock
+} from "@openzeppelin/contracts/governance/extensions/GovernorTimelockCompound.sol";
+import {IERC20} from "@openzeppelin/contracts/interfaces/IERC20.sol";
+
+import {Propose} from "script/Propose.s.sol";
+import {IGovernorAlpha} from "src/interfaces/IGovernorAlpha.sol";
+import {PooltogetherGovernorTest} from "test/helpers/PooltogetherGovernorTest.sol";
+
+abstract contract AlphaProposalTest is PooltogetherGovernorTest {
+  //----------------- State and Setup ----------- //
+
+  IGovernorAlpha governorAlpha = IGovernorAlpha(GOVERNOR_ALPHA);
+  IERC20 usdcToken = IERC20(USDC_ADDRESS);
+  IERC20 radToken = IERC20(RAD_ADDRESS);
```

Comment:
We can remove this token (or both of them for now, until we get to testing them)

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/11/commits/33df525e4b15b3c0126dfa3caa34171e77cd36a8